src/components/molecules/modals/FollowModal/index.jsx

```diff
@@ -0,0 +1,238 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+// DOM bindings for React Router
+import { Link } from "react-router-dom";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBModal,
+  MDBIcon,
+  MDBModalBody,
+  MDBBtn,
+  MDBModalHeader,
+  MDBRow,
+  MDBCol,
+} from "mdbreact";
+
+//> Stylesheet
+import "./follow.scss";
+//#endregion
+
+//#region > Components
+/** @class A modal that shows a list of follows */
+class FollowModal extends React.Component {
+  state = {
+    loggedUser: null,
+  };
```

Comment:
I think it would be better not to store the whole loggedUser into the state.
We should load only required datasets into the state.